tests/autotest.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
```

Comment:
This file should be removed. I don't suppose it works even for device manager by now.
(furthermore, if anyone is to do this sort of automatic testing, it's Travis, not Jenkins)